javascript-modal/main.js

```diff
@@ -0,0 +1,13 @@
+var open = document.querySelector('.open-button');
+var modal = document.querySelector('.modal');
+var close = document.querySelector('.modal-button');
+
+function overlay(event) {
+  modal.style.display = 'block';
```

Comment:
Oh noooo, this is inline styling, exactly the same as using the `style` attribute on an element 💀 

Instead, I would make a class that you can add or remove to hide/show the element 🤓